packages/core/data-transfer/src/strapi/providers/local-destination/strategies/restore/links.ts

```diff
@@ -7,7 +7,8 @@ import { createLinkQuery } from '../../../../queries/link';
 export const createLinksWriteStream = (
   mapID: (uid: string, id: number) => number | undefined,
   strapi: LoadedStrapi,
-  transaction?: Transaction
+  transaction?: Transaction,
+  onWarning?: any
```

Comment:
we can have the actual type instead of any
```suggestion
  onWarning?: ((message: string) => void)
```